lib/view/page/splash_page.dart

```diff
@@ -31,11 +37,23 @@ class _SplashPageState extends State<SplashPage> {
     /// Pass all uncaught errors from the framework to Crashlytics.
     FlutterError.onError = FirebaseCrashlytics.instance.recordFlutterError;
     debugPrint('Firebase Crashlytics flutter errors enabled...');
-    
+
     SchedulerBinding.instance.addPostFrameCallback((timeStamp) {
       // ensure the onComplete callback cannot be called in the same frame
       widget.onComplete();
     });
+
+    AuthenticationService _authenticationService = new AuthenticationService();
+    IdeaService _ideaService = new IdeaService();
+    SprintService _sprintService = new SprintService();
+    UserService _userService = new UserService();
```

Comment:
Services should be registered as singletons in locator.dart

Reply:
What does this mean?